perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -103,7 +103,7 @@ def _PushRobustCommandScripts(self):
                                             os.path.basename(f)))
         self._has_remote_command_script = True
 
-  def RobustRemoteCommand(self, command):
+  def RobustRemoteCommand(self, command, should_log=False):
```

Comment:
I didn't add timeout to this change just because I don't need it right now and I wanted to keep this PR as simple as possible. If you want, I'll start a new PR and add timeouts, and maybe other features of RemoteCommand().